### ISSUE TITLE
Upgrading from 6.0.3 to 6.1.0  doesn't work

### ISSUE THREAD

**Issue by AdrianXIA** [oboukili/terraform-provider-argocd#382]
Hello!

Upgrading the ArgoCD Terraform provider from 6.0.3  to 6.1.0.  I get the following error: 
![image](https://github.com/oboukili/terraform-provider-argocd/assets/7112324/123d9262-7cb8-4a7a-bff4-16ecbc994648)

I cannot figure out what I should update to fix the issue.
Do you have any hints, please?

### Terraform Version, ArgoCD Provider Version and ArgoCD Version

```
Terraform version: 1.7.4
ArgoCD provider version: 6.1.0
ArgoCD version: 2.10.2
```

**Comment by the-technat** [comment]
That should be fixed with #378

**Comment by onematchfox** [comment]
Fixed in [v6.1.1](https://github.com/oboukili/terraform-provider-argocd/releases/tag/v6.1.1)
